srcpkgs/exa/template

```diff
@@ -1,21 +1,24 @@
 # Template file for 'exa'
 pkgname=exa
-version=0.9.0
-revision=2
+version=0.10.0
+revision=1
 build_style=cargo
-hostmakedepends="pkg-config"
+hostmakedepends="pkg-config libgit2-devel"
 makedepends="libgit2-devel"
 short_desc="Modern replacement for ls"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="FollieHiyuki <folliekazetani@protonmail.com>"
 license="MIT"
 homepage="https://the.exa.website/"
-distfiles="https://github.com/ogham/exa/archive/v${version}.tar.gz"
-checksum=@facbe3b234f403e1e1919f28f009f1733a0fbbbb11f0bc2ba89ac3eba11bc5e8
+distfiles="https://github.com/ogham/exa/archive/v${version}.tar.gz
+ https://github.com/ogham/exa/releases/download/v${version}/exa-accoutrements-v${version}.zip"
```

Comment:
The only issue is that this means we have a leftover directory when the package is cleaned up.

You can move it inside the wrksrc in `post_extract` doing: `mv ../exa-accoutrements-v$version accoutrements` and then `vinstall` later with the new path.

Reply:
Done